### PR TITLE
Actualizando buy-shirt.spec.ts,Cypress.json, y los package.json

### PR DESCRIPTION
**Pull request by Antonieta13**
Step 8 CSS Selectors

### CODE REVIEW

File: buy-shirt.spec.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        cy.get('#email').type('aperdomobo@gmail.com')
+        cy.get('#passwd').type('WorkshopProtractor')
+
+        // Debes completar la prueba ...
+
+        cy.get('#center_column > div > p > strong')
+            .should('have.text', 'Your order on My Store is complete.')
+    });
+});
+
```
Comment (holgiosalos) [inline buy-shirt.spec.ts]: delete spaces in line 20

File: buy-shirt.spec.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
```
Comment (holgiosalos) [inline buy-shirt.spec.ts]: delete this line

Comment (Antonieta13) [reply buy-shirt.spec.ts]: Ready

File: buy-shirt.spec.ts (hunk 1 above)
Comment (Antonieta13) [reply buy-shirt.spec.ts]: Ready

**Review by holgiosalos — APPROVED** [review]

File: buy-shirt.spec.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        cy.visit('http://automationpractice.com/')
+        cy.get('#block_top_menu > ul > li:nth-child(3) > a').click()
+        cy.get('#center_column a.button.ajax_add_to_cart_button.btn.btn-default').click()
+        cy.get('[style*="display: block;"] .button-container > a').click()
+        cy.get('.cart_navigation span').click()
+
+        cy.get('#email').type('aperdomobo@gmail.com')
+        cy.get('#passwd').type('WorkshopProtractor')
+
+        // Debes completar la prueba ...
```
Comment (renardete) [inline buy-shirt.spec.ts]: you must complete the test, using the gif in the trainning documentation.

**Review by renardete — DISMISSED** [review]

File: buy-shirt.spec.ts (hunk 3 above)
Comment (Antonieta13) [reply buy-shirt.spec.ts:13]: Juan, I should open http://automationpractice.com/ from cypress when this opens google? I mean I try this but google open a list but when do click to http://automationpractice.com/ doesn't work

Comment (Antonieta13) [reply buy-shirt.spec.ts:13]: I opened google since my google chrome navigator and work correctly :). What I should do?

**Review by holgiosalos — CHANGES_REQUESTED** [review]
Please locate the `buy-shirt.spec.ts` under `cypress/integration/` folder.

Also, you should complete the tests by adding the needed actions to finish the order. I will put a comment on slack channel :)

File: buy-shirt.spec.ts (hunk 3 above)
Comment (Antonieta13) [reply buy-shirt.spec.ts]: I finish the test! Finally :)

**Comment by Antonieta13** [comment]
Hi, Guys I know it's Sunday, but finally, I finish step 8 CSS Selector. This takes me a lot of time but is my first time working on that. 
Waiting for your approvals

Thank you :)

**Review by holgiosalos — APPROVED** [review]
Great job!

File: buy-shirt.spec.ts (hunk 3 above)
Comment (Antonieta13) [reply buy-shirt.spec.ts]: I'm still waiting for your review and approval Pleaseeee! @renardete

File: cypress/integration/buy-shirt.spec.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        cy.visit("http://automationpractice.com/");
+        cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+        cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
+        cy.get("[style*=\"display: block;\"] .button-container > a").click();
+        cy.get(".cart_navigation span").click();
+
+        cy.get("#email").type("aperdomobo@gmail.com");
+        cy.get("#passwd").type("WorkshopProtractor");
+
+        // Debes completar la prueba ...
```
Comment (renardete) [inline cypress/integration/buy-shirt.spec.ts]: delete the unnecessary comment

Comment (Antonieta13) [reply cypress/integration/buy-shirt.spec.ts]: Ready

**Pull request merged**